testing/srnca/test_nca.py

```diff
@@ -16,7 +16,7 @@ def test_count_parameters(self):
 
         for hidden in [2,8,32,64]:
             for channels in [1,3,5,15]:
-                for filters in [3,4,5]:
+                for filters in [3,4,5, 15]:
```

Comment:
test for additional filters beyond those that are hand-coded (id, laplacian, soble_h, sobel_w, moore). additional filters are sampled from a random uniform distribution and concatenated to the filters stack. choosing fewer filters truncates the filters used.